backend/minors/AAS.yaml

```diff
@@ -0,0 +1,515 @@
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 352
+    - LAS 356
+    - AAS 352
+    - SPA 363
+    - LAS 334
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 366
+    - AMS 326
+    - LAO 366
+    - GSS 364
+    - SPA 387
+    - AAS 387
+    - LAO 387
+    - LAS 381
+    - SPA 408
+    - LAS 418
+    - MUS 338
+    - THR 355
+    - AAS 399
+    - THR 355
+    - AAS 399
+    - AFS 353
+    - URB 305
+    - SAS 351
+    - AAS 364
+    - ARC 325
+    - VIS 233
+    - AAS 233
+    - VIS 234
+    - AAS 234
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 352
+    - LAS 356
+    - AAS 352
+    - SPA 363
+    - LAS 334
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 366
+    - AMS 326
+    - LAO 366
+    - GSS 364
+    - SPA 387
+    - AAS 387
+    - LAO 387
+    - LAS 381
+    - SPA 408
+    - LAS 418
+    - MUS 338
+    - THR 355
+    - AAS 399
+    - THR 355
+    - AAS 399
+    - AFS 353
+    - URB 305
+    - SAS 351
+    - AAS 364
+    - ARC 325
+    - VIS 233
+    - AAS 233
+    - VIS 234
+    - AAS 234
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 352
+    - LAS 356
+    - AAS 352
+    - SPA 363
+    - LAS 334
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 366
+    - AMS 326
+    - LAO 366
+    - GSS 364
+    - SPA 387
+    - AAS 387
+    - LAO 387
+    - LAS 381
+    - SPA 408
+    - LAS 418
+    - MUS 338
+    - THR 355
+    - AAS 399
+    - THR 355
+    - AAS 399
+    - AFS 353
+    - URB 305
+    - SAS 351
+    - AAS 364
+    - ARC 325
+    - VIS 233
+    - AAS 233
+    - VIS 234
+    - AAS 234
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 233
+    - LIN 233
+    - LAS 233
+    - SPA 352
+    - LAS 356
+    - AAS 352
+    - SPA 363
+    - LAS 334
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 365
+    - LAO 365
+    - URB 365
+    - SPA 366
+    - AMS 326
+    - LAO 366
+    - GSS 364
+    - SPA 387
+    - AAS 387
+    - LAO 387
+    - LAS 381
+    - SPA 408
+    - LAS 418
+    - MUS 338
+    - THR 355
+    - AAS 399
+    - THR 355
+    - AAS 399
+    - AFS 353
+    - URB 305
+    - SAS 351
+    - AAS 364
+    - ARC 325
+    - VIS 233
+    - AAS 233
+    - VIS 234
+    - AAS 234
+  - name: Restrictions
+  max_counted: 0
+  min_needed: 0
+  explanation: |-
+    Students may count one course taken for their major toward the minor if the course 
+    is cross-listed with AAS or has been approved as a cognate by the AAS Director of 
+    Undergraduate Studies before the course is taken. The PDF option counts toward the minor
+    if the course is primary-listed with AAS, cross-listed with AAS, or has been approved as 
+    a creative or practice-based cognate by the AAS Director of Undergraduate Studies before
```

Comment:
Be sure to add an editor's note, and make `max_counted` and `min_needed` 1. For reference, checkout [URB.yaml](https://github.com/HoagieClub/plan/blob/b4d517df2311073ea79317b52f8163e0cac9f5f3/backend/certificates/URB.yaml)